src/structures/Guild.js

```diff
@@ -1422,6 +1476,38 @@ class Guild extends Base {
       .then(() => this);
   }
 
+  /**
+   * Edits the guild's membership screening form.
+   * @param {Partial<GuildMembershipScreening>} memberScreen The membership screening data for the guild
+   * @returns {Promise<GuildMembershipScreening>}
+   */
+  async setMembershipScreening(memberScreen) {
+    if (!this.features.includes('COMMUNITY')) {
+      throw new Error('COMMUNITY');
+    }
+    const fields = memberScreen.formFields?.map(field => ({
+      field_type: field.field_type,
+      label: field.label,
+      values: field.values,
+      required: field.required,
+    }));
+    const data = {};
+    if (typeof memberScreen.enabled !== 'undefined') data.enabled = memberScreen.enabled;
+    if (typeof memberScreen.description !== 'undefined') data.description = memberScreen.description;
+    if (typeof memberScreen.formFields !== 'undefined') data.form_fields = JSON.stringify(fields);
```

Comment:
```suggestion
    if (typeof memberScreen.formFields !== 'undefined') data.form_fields = fields;
```
i don't think this is meant to be stringified here

Reply:
In my testing (and according to the referenced docs) it has to be stringified. This may be an accident in the API, but this currently accurately reflects what the API is expecting.

Reply:
oops, misread docs mb